File: 30DaysOfCode/day06_review/letsReview.pl
$T = <STDIN>;
chomp $T;

for ($i = 0; $i < $T; $i++) {
    $S = <STDIN>;
    chomp $S;
    
    for ($j = 0; $j < length($S); $j += 2) {
        print substr($S, $j, 1);
    }
    print " ";
    
    for ($j = 1; $j < length($S); $j += 2) {
        print substr($S, $j, 1);
    }
    print "\n";
}

File: 30DaysOfCode/day06_review/letsReview.php
<?php
$_fp = fopen("php://stdin", "r");

$N = fgets($_fp);

for ($i = 0; $i < $N; $i++) {
    $x = trim(fgets($_fp));
    
    for ($j = 0; $j < strlen($x); $j += 2) {
        printf(substr($x, $j, 1));
    }
    printf(" ");
    
    for ($j = 1; $j < strlen($x); $j += 2) {
        printf(substr($x, $j, 1));
    }
    printf("\n");
}
?>
